yt/yt/library/auth_server/cypress_user_manager.h

```diff
@@ -14,6 +14,7 @@ struct ICypressUserManager
     : public virtual TRefCounted
 {
     virtual TFuture<NObjectClient::TObjectId> CreateUser(const TString& name) = 0;
+    virtual TFuture<bool> UserExists(const TString& name) = 0;
```

Comment:
Let's use a verb in the name, since we have `CreateUser` already. Like `CheckUserExists`.